src/checkout/api_default.go

```diff
@@ -14,41 +14,41 @@ import (
 	_context "context"
 	_nethttp "net/http"
 
-	"github.com/adyen/adyen-go-api-library/v4/src/common"
+	"github.com/adyen/adyen-go-api-library/v5/src/common"
 )
 
 // Checkout Checkout service
 type Checkout common.Service
 
 /*
-GetPaymentLink Retrieve a payment link.
+PaymentLinks Retrieve a payment link.
 Retrieves the payment link details using the payment link &#x60;id&#x60;.
  * @param linkId Unique identifier of the payment link.
  * @param ctxs ..._context.Context - optional, for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
 @return PaymentLinkResource
 */
-func (a Checkout) GetPaymentLink(linkId string, ctxs ..._context.Context) (PaymentLinkResource, *_nethttp.Response, error) {
+func (a Checkout) GetPaymentLinks(linkId string, ctxs ..._context.Context) (PaymentLinkResource, *_nethttp.Response, error) {
 	res := &PaymentLinkResource{}
 	httpRes, err := a.Client.MakeHTTPGetRequest(res, a.BasePath()+"/paymentLinks/"+linkId, ctxs...)
 	return *res, httpRes, err
 }
 
 /*
-UpdatePaymentLink Update the status of a payment link
-Updates the status of a payment link. Use this endpoint to [force the expiry of a payment link](https://docs.adyen.com/checkout/pay-by-link#update-payment-link-status).
+UpdatePaymentLinks Update the status of a payment link
+Updates the status of a payment link. Use this endpoint to [force the expiry of a payment link](https://docs.adyen.com/online-payments/pay-by-link#update-payment-link-status).
  * @param linkId Unique identifier of the payment link.
  * @param request UpdatePaymentLinkRequest - reference of UpdatePaymentLinkRequest).
  * @param ctxs ..._context.Context - optional, for authentication, logging, cancellation, deadlines, tracing, etc. Passed from http.Request or context.Background().
 @return PaymentLinkResource
 */
-func (a Checkout) UpdatePaymentLink(linkId string, req *UpdatePaymentLinkRequest, ctxs ..._context.Context) (PaymentLinkResource, *_nethttp.Response, error) {
+func (a Checkout) UpdatePaymentLinks(linkId string, req *UpdatePaymentLinkRequest, ctxs ..._context.Context) (PaymentLinkResource, *_nethttp.Response, error) {
```

Comment:
the same here, we only update 1 paymentLink based on the linkId.

Reply:
same as above